src/modules/name/index.ts

```diff
@@ -1,30 +1,43 @@
 import type { Faker } from '../..';
 import { deprecated } from '../../internal/deprecated';
 
-// disabled until renamed to Sex
+/**
+ * @deprecated
```

Comment:
In our [deprecation workflow documentation](https://github.com/faker-js/faker/blob/main/CONTRIBUTING.md#deprecation-workflow) we require a human-readable message for the `deprecated` JSDoc parameter.

```suggestion
 * @deprecated Use Sex enum instead.
```